openedxstats/apps/sites/templates/sites/sites_list.html

```diff
@@ -67,8 +88,15 @@
                      { "data": "fields.active_start_date" },
                      { "data": "fields.active_end_date" },
                      { "data": "fields.isCurrentVersion" },
-                     { "data": "fields.actionButtons" }
-                ]
+                     { "data": "fields.actionButtons" },
+                     { "data": "fields.geolocation",
+                        "visible": false },
+                    { "data": "fields.notes",
+                        "visible": false },
+                    { "data": "fields.language",
+                        "visible": false
+                    }
```

Comment:
The indentation in these lines is all over the place.
I think it would be easier to read if you kept "data" and "visible" on the same line:
```
    { "data": "fields.actionButtons" },
    { "data": "fields.geolocation", "visible": false },
    { "data": "fields.notes", "visible": false },
```